contracts/helpers/MultiBlockHarvester.sol

```diff
@@ -129,6 +131,7 @@ contract MultiBlockHarvester is BaseHarvester {
                     address(this),
                     block.timestamp
                 );
+                _checkSlippage(shares, amountOut, yieldBearingAsset, depositAddress, true);
```

Comment:
let's jsut do one check, we should remove the one at line 121

And theredore add one in below condiiton USDM